gravityflow/gravityflow
# Display inbox count in the WP Dashboard

## Description
Added the red bubble inbox count display with the Workflow Label on the WP Dashboard

## Testing instructions
Switch to this branch and confirm the presence of the 'red bubble' with the inbox entries count next to the 'Workflow' on the WP Dashboard.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes?
N/A

## Checklist:
